src/middleware/github-webhook-middleware.ts

```diff
@@ -84,7 +87,10 @@ export const GithubWebhookMiddleware = (
 			event: webhookEvent,
 			action: context.payload?.action,
 			id: context.id,
-			repo: context.payload?.repository ? context.repo() : undefined,
+			repo: context.payload?.repository ? {
+				owner: context.payload.repository.owner.login,
```

Comment:
This was context.repo() before. Won't this break if we're using the current context?

Reply:
context.repo() was just a utility function to get the repo information out from the payload in a certain format :)